src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  width: 100%;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.logo-img {
+  padding-top: 2px;
+}
+
+.navigation {
+  width: 635.54px;
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+}
```

Comment:
why did you reset padding in the class selector, and margin in the tag selector for the same element?